xlators/storage/posix/src/posix-inode-fd-ops.c

```diff
@@ -6020,10 +6020,7 @@ posix_readdirp(call_frame_t *frame, xlator_t *this, fd_t *fd, size_t size,
         if (op_ret >= 0) {
             op_ret = 0;
 
-            list_for_each_entry(entry, &entries.list, list)
-            {
-                op_ret++;
-            }
+            list_for_each_entry(entry, &entries.list, list) { op_ret++; }
```

Comment:
I wouldn't do this change. We never put short code blocks in a single line.

Reply:
Yeah, but I don't think the clang-format check is going to succeed without this change. I see couple of other files having this syntax now, may be implemented due to the similar formatting suggestion from clang.

Reply:
I think our clang-format rules are too outdated and it's proposing changes that initially were not intended (and it didn't show). Probably it's time to review it.

In any case clang-format check is not enforced, it's just informative and patches can be merged even if clang-format says something.

Reply:
Ahh!! okay. Since the clang-format check is marked as required, I have applied the change to make it happy :). I will revert that change.